src/Gui/CMakeLists.txt

```diff
@@ -78,79 +78,70 @@ else(MSVC)
-        ${Qt5PrintSupport_INCLUDE_DIRS}
-        ${Qt5Svg_INCLUDE_DIRS}
-        ${Qt5Network_INCLUDE_DIRS}
-        ${Qt5UiTools_INCLUDE_DIRS}
-    )
-    list(APPEND FreeCADGui_LIBS
-        ${Qt5Core_LIBRARIES}
-        ${Qt5Widgets_LIBRARIES}
-        ${Qt5OpenGL_LIBRARIES}
-        ${Qt5PrintSupport_LIBRARIES}
-        ${Qt5Svg_LIBRARIES}
-        ${Qt5Network_LIBRARIES}
-        ${Qt5UiTools_LIBRARIES}
-    )
-    if(${Qt5WinExtras_FOUND})
-      include_directories(
         ${Qt5WinExtras_INCLUDE_DIRS}
-      )
-      list(APPEND FreeCADGui_LIBS
-        ${Qt5WinExtras_LIBRARIES}
-      )
-    endif()
-else()
-    include_directories(
-        ${QT_INCLUDE_DIR}
     )
     list(APPEND FreeCADGui_LIBS
-        ${QT_LIBRARIES}
-        ${QT_QTUITOOLS_LIBRARY}
+        ${Qt5WinExtras_LIBRARIES}
     )
 endif()
 
 IF(SPNAV_FOUND)
     if(SPNAV_USE_X11)
-      add_definitions(-DSPNAV_USE_X11)
-      if (BUILD_QT5 AND UNIX AND NOT APPLE)
-        find_package(Qt5X11Extras REQUIRED)
-        include_directories(${Qt5X11Extras_INCLUDE_DIRS})
-        list(APPEND FreeCADGui_LIBS ${Qt5X11Extras_LIBRARIES})
-      endif()
-      find_package(X11 QUIET)
-      if (X11_FOUND)
-        list(APPEND FreeCADGui_LIBS
-            ${X11_X11_LIB}
-        )
-      endif(X11_FOUND)
-      SET(FreeCADGui_SDK_SRCS
-        3Dconnexion/GuiAbstractNativeEvent.cpp
-        3Dconnexion/GuiNativeEventLinuxX11.cpp
-        )
-      SET(FreeCADGui_SDK_MOC_HDRS
-        3Dconnexion/GuiAbstractNativeEvent.h
-        3Dconnexion/GuiNativeEventLinuxX11.h
-        )
+        add_definitions(-DSPNAV_USE_X11)
+        if (FREECAD_QT_MAJOR_VERSION EQUAL 5 AND UNIX AND NOT APPLE)
+            find_package(Qt5X11Extras REQUIRED)
+            include_directories(${Qt5X11Extras_INCLUDE_DIRS})
+            list(APPEND FreeCADGui_LIBS ${Qt5X11Extras_LIBRARIES})
+            # Note that Qt6 has removed the Qt*Extras files
+        endif()
+        find_package(X11 QUIET)
+        if (X11_FOUND)
+            list(APPEND FreeCADGui_LIBS
+                ${X11_X11_LIB}
+            )
+        endif(X11_FOUND)
+        SET(FreeCADGui_SDK_SRCS
+            3Dconnexion/GuiAbstractNativeEvent.cpp
+            3Dconnexion/GuiNativeEventLinuxX11.cpp
+            )
+        SET(FreeCADGui_SDK_MOC_HDRS
+            3Dconnexion/GuiAbstractNativeEvent.h
+            3Dconnexion/GuiNativeEventLinuxX11.h
+            )
     else(SPNAV_USE_X11)
-      SET(FreeCADGui_SDK_SRCS
-        3Dconnexion/GuiAbstractNativeEvent.cpp
-        3Dconnexion/GuiNativeEventLinux.cpp
-      )
-      SET(FreeCADGui_SDK_MOC_HDRS
-        3Dconnexion/GuiAbstractNativeEvent.h
-        3Dconnexion/GuiNativeEventLinux.h
-      )
+        SET(FreeCADGui_SDK_SRCS
+            3Dconnexion/GuiAbstractNativeEvent.cpp
+            3Dconnexion/GuiNativeEventLinux.cpp
+            )
+        SET(FreeCADGui_SDK_MOC_HDRS
+            3Dconnexion/GuiAbstractNativeEvent.h
+            3Dconnexion/GuiNativeEventLinux.h
+            )
```

Comment:
This looks like you are using mixed whitespace (tabs, spaces).

Reply:
I'm asking because of different coloring in the browser. I see the light green background and in in front of the text in these lines there's a slightly darker green. I saw these on other PRs, where it was mixed spacing. Maybe it's only the change in indentation here. Sorry for the noise.

Reply:
Got it -- yes, in GitHub added whitespace of any kind shows up like that -- to actually see if it's a tab you can either try to select it, or look at the CI output (which complains about mixing tabs and spaces if you do it).